src/renderer/kube-object.store.ts

```diff
@@ -11,7 +11,8 @@ import { getHostedCluster } from "../common/cluster-store";
 @autobind()
 export abstract class KubeObjectStore<T extends KubeObject = any> extends ItemStore<T> {
   abstract api: KubeApi<T>;
-  public limit: number;
+  public readonly abstract limit: number;
```

Comment:
We could call limit `requestLimit`

Reply:
Changing name would be a breaking change.. even `abstract` is breaking change (CI shows failures).

Reply:
@jakolehm Removed `abstract` can you review again, please?